### PR TITLE
fixes unit for performance data

### LINKED ISSUE #7185: Storage performance metrics writeTotalSecs
The value of writeTotalSecs is somthing like 14585200 seconds that is 4051 hours. The number did not make any sense. @amaltaro, @ticoann  can you take a look below code?
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/FwkJobReport/XMLParser.py#L448

### PR DESCRIPTION
**Pull request by ticoann**
fixes #7185 
@yuyiguo, @juztas, @amaltaro please review.
I change other units  from msec to sec. However this implies the keyword changes in fwjr data.

readMaxMSec -> readMaxSec

I am not sure whether this will break anything or it is too big of a unit. If anyone concerned I can change back to MSec.

### CODE REVIEW

**Comment by amaltaro** [comment]
If we retrieve the Msec information from the original fwjr, then I'd be in favor of keeping the same unit for WMArchive

**Comment by juztas** [comment]
looks ok for me

File: src/python/WMCore/FwkJobReport/XMLParser.py [hunk 1]
```diff
@@ -433,24 +433,24 @@ def perfStoreHandler():
...
-            readMaxMSec = max(storageValues.get("Timing-%s-read-maxMsecs" % readMethod, 0),
-                              storageValues.get("Timing-%s-readv-maxMsecs" % readMethod, 0))
+            readMaxSec = (max(storageValues.get("Timing-%s-read-maxMsecs" % readMethod, 0),
+                              storageValues.get("Timing-%s-readv-maxMsecs" % readMethod, 0))) // 1000
             readPercOps = storageValues.get("Timing-tstoragefile-readActual-numOperations", 0)/\
                           storageValues.get("Timing-tstoragefile-read-numOperations", 0)
             readCachOps = storageValues.get("Timing-tstoragefile-readViaCache-numSuccessfulOperations", 0)/\
                           storageValues.get("Timing-tstoragefile-read-numOperations", 0)
-            readTotalT  = 1000 * storageValues.get("Timing-tstoragefile-read-totalMSecs", 0)
+            readTotalT  = storageValues.get("Timing-tstoragefile-read-totalMSecs", 0) // 1000
```
Comment (yuyiguo) [inline src/python/WMCore/FwkJobReport/XMLParser.py]: "//1000" or "/1000"?

Comment (ticoann) [inline src/python/WMCore/FwkJobReport/XMLParser.py]: // is integer division if we keep the sec probably we don't want get so precise numbers.

**Comment by ticoann** [comment]
Alan, I agree with you. Anyway, Yuyi will provide some statistics which unit make more sense. (milisec or sec for all the parameters). And we can update the patch.

**Comment by yuyiguo** [comment]
MongoDB was having problem that prevented index creation and searching w/o indexes was too slow to finish.  I think using minutes are safe.

**Comment by ticoann** [comment]
@amaltaro, @vkuznet, @yuyiguo, @juztas, As we discussed in the meeting we will keep the current keyword. readMaxMSec and still the unit is millisecond. Please review. If everyone is ok with this change I will patch the production agents.

**Comment by amaltaro** [comment]
test this please

**Comment by amaltaro** [comment]
I'm backporting this fix to the next agent release, however this one has to have the unittest fixed before it gets merged (testPerformanceSummary)

**Comment by amaltaro** [comment]
Seangchan, it was backported to the wmagent branch.
@ticoann feel free to merge it (I'm removing the labels)

**Comment by amaltaro** [comment]
I almost forgot that this test has to be fixed:
WMCore_t.FwkJobReport_t.Report_t.ReportTest:testPerformanceSummary changed from success to failure

**Pull request merged**